<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueryDialog</class>
 <widget class="QDialog" name="QueryDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="9" column="0">
    <widget class="QGroupBox" name="groupGroups">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Поля Группировок</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0" colspan="2">
       <widget class="CRCTableFieldsView" name="tableGroups"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QGroupBox" name="groupOrders">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Поля для сортировки</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="1" column="0" colspan="2">
       <widget class="CRCTableFieldsView" name="tableOrders"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QGroupBox" name="groupConds">
     <property name="title">
      <string>Условия</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" colspan="2">
       <widget class="CRCConditionsTreeView" name="treeConds">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupFields">
     <property name="title">
      <string>Поля таблицы</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0" colspan="2">
       <widget class="CRCTableFieldsView" name="tableFields"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblName">
       <property name="text">
        <string>Наименование</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtName"/>
     </item>
     <item>
      <widget class="QLabel" name="lblMainTable">
       <property name="text">
        <string>Главная таблица</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CRCComboBox" name="cmbMainTable">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblReference">
       <property name="text">
        <string>Ссылка на поле</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CRCFieldsComboBox" name="cmbReference">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRCComboBox</class>
   <extends>QComboBox</extends>
   <header>Reports/ReportsConstructor/RCComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>CRCConditionsTreeView</class>
   <extends>QTreeView</extends>
   <header>Reports/ReportsConstructor/RCTreeView.h</header>
  </customwidget>
  <customwidget>
   <class>CRCTableFieldsView</class>
   <extends>QTableView</extends>
   <header>Reports/ReportsConstructor/RCTableView.h</header>
  </customwidget>
  <customwidget>
   <class>CRCFieldsComboBox</class>
   <extends>QComboBox</extends>
   <header>Reports/ReportsConstructor/RCComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QueryDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QueryDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
